protos/sigstore_verification.proto

```diff
@@ -54,6 +54,12 @@ message ArtifactVerificationOptions {
                 bool perform_online_verification = 2;
                 // Disable verification for transparency logs.
                 bool disable = 3;
+                // Verify SET indicates that the timestamp from
+                // the SET should be used when verifying the X.509
+                // certifiacte chain. Note that inclusion promises (SETs)
+                // are optional in a bundle, and so requiring SET may render
+                // bundles invalid during verification if SET is not present.
+                bool verify_set = 4;
```

Comment:
This is an extreme nitpick, but IMO the comment and the field name are slightly out of sync here: the comment says that the SET's embedded timestamp is *used to* fill in the X.509 chain's validation time, whereas the field name implies that the option enables verifying the SET itself.

Maybe `verify_time_from_set` or similar?

Reply:
Yeah, it was first called `verify_set_timestamp = 4;` but changed based on review. Let's keep this open until the PR in `sigstore-go` https://github.com/sigstore/sigstore-go/issues/44 is resolved, where this exact semantics is being discussed. I think we should align the naming between them.